File: src/main/java/com/example/movieapi/web/rest/AdminController.java
package com.example.movieapi.web.rest;

import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;

@Slf4j
@Controller
@RequiredArgsConstructor
public class AdminController {

    @GetMapping("/admin")
    public String admin() {
        return "/admin/admin";}
}

File: src/main/java/com/example/movieapi/web/rest/AdminController.kt
package com.example.movieapi.web.rest

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.ResponseBody

@Controller
@ResponseBody
class AdminController {

    @GetMapping("/admin")
    fun adminP(): String {
        return "Admin Controller"
    }
}
